src/react_native/gesture.cljs

```diff
@@ -112,17 +113,21 @@
            render-fn]
     :or   {sticky-section-headers-enabled true}
     :as   props}]
-  (let [data (flatten-sections sections)]
+  (let [data        (flatten-sections sections)
+        render-item (rn/use-callback
+                     (fn [p1 p2 p3 p4]
```

Comment:
What is `p4` if there are only 3 deps?

Reply:
So I think `p1` `p2` `p3` `p4` are the arguments that are passed to the render function. So in this case `p1` is the item in the list, `p2` is the index, `p3` is a reference to the separators in the list, and `p4` is the render-data that's passed from the list component.

More details can be found here: https://github.com/status-im/status-mobile/blob/afe90d0ec989c23e0bb3b41725fa67366fe721e4/src/react_native/flat_list.cljs#L8-L13

Reply:
Thanks, my brain farted and was confused with `use-callback`.